### ISSUE TITLE
Crash on S3 bucket import

### ISSUE THREAD

**Issue by nwalke** [terraform-providers/terraform-provider-aws#2194]
### Terraform Version
0.10.8

### Affected Resource(s)
`aws_s3_bucket`

### Terraform Configuration Files
```hcl
resource "aws_s3_bucket" "database_wal_segments" {
  bucket = "${var.database_wal_segments_bucket_name}"
  acl = "private"

  tags {
    Name = "${var.database_wal_segments_bucket_name}"
    terraform = "true"
  }
}
```

### Debug Output
https://gist.github.com/nwalke/1630d5cfb65d64eed4def3eaf43a8e7f

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
The S3 bucket would be imported.

### Actual Behavior
Terraform crashed while refreshing state.

### Steps to Reproduce
All I did was:
1. `terraform import module.mymodule.aws_s3_bucket.database_wal_segments mybucket`

### Important Factoids
We assume a role with the AWS provider.  Hopefully it's using that to do imports.

### References
Filed this issue on core since it was able to crash Terraform: https://github.com/hashicorp/terraform/issues/16574

**Comment by kjmkznr** [comment]
I think, you use terraform-provider-aws-0.1.3.
This problem fixes in #1316.

You should upgrade terraform-provider-aws-0.1.4 or later.

```
provider "aws" {
  version = "0.1.4"
}
```
or
```
provider "aws" {
  version = "1.2.0"
}
```

**Comment by nwalke** [comment]
Oh wow, yeah.  My provider was way out of date.  Sorry about that!
